diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/ClassLikeStructuresOrderRule.kt

```diff
@@ -61,10 +61,12 @@ class ClassLikeStructuresOrderRule(private val configRules: List<RulesConfig>) :
         val allProperties = node.getChildren(TokenSet.create(PROPERTY))
         val constProperties = allProperties.filter { it.findLeafWithSpecificType(CONST_KEYWORD) != null }.toMutableList()
         val lateInitProperties = allProperties.filter { it.findLeafWithSpecificType(LATEINIT_KEYWORD) != null }.toMutableList()
-        val loggers = allProperties.filter {
-            (it.findChildByType(MODIFIER_LIST) == null || it.findLeafWithSpecificType(PRIVATE_KEYWORD) != null)
-                    && it.getIdentifierName()!!.text.contains(loggerPropertyRegex)
-        }.toMutableList()
+        val loggers = allProperties
```

Comment:
I guess we will need to change this logic:
 allProperties.filter
.foo()
.bar()

is valid

Reply:
So like if first call in chain is multiline lamba, then it should start on the same line as initial receiver?

Reply:
that should be possible, not prohibited

Reply:
Create #326 for this